spec/controllers/idv/capture_doc_controller_spec.rb

```diff
@@ -32,7 +32,7 @@
     end
 
     context 'with no session' do
-      it 'redirects to the root url' do
+      xit 'redirects to the root url' do
```

Comment:
how come these were disabled? do we intend to leave them disabled?

Reply:
nvm, asked & answered: https://github.com/18F/identity-idp/pull/8325#issuecomment-1533753094

Reply:
Disabled because I broke them by changing the routes. They will be deleted in #8292. This is getting merged separately to avoid redirecting people to missing code during the 50/50 state.